### PR TITLE
Remove serve command

### PR DESCRIPTION
**Pull request by marknguyen1302**
## Describe Your Changes

- remove the serve command
- change the serve command to root
- require server online for every command
- detach mode by default

## Fixes Issues

- Closes #
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

**Review by namchuai — APPROVED** [review]

File: cortex-js/src/infrastructure/commanders/cortex-command.commander.ts [hunk 1]
```diff
@@ -38,15 +44,66 @@ import { ConfigsCommand } from './configs.command';
...
+  @Option({
+    flags: '-p, --port <port>',
+    description: 'Port to serve the application',
+  })
+  parsePort(value: string) {
+    return parseInt(value, 10);
+  }
+
+  @Option({
+    flags: '-A, --attach',
```
Comment (louis-jan) [inline cortex-js/src/infrastructure/commanders/cortex-command.commander.ts:103]: to confirm the syntax..

**Comment by louis-jan** [comment]
Should not log JS endpoint

Screenshot 2024-07-22 at 09 48 37

**Comment by louis-jan** [comment]
Start with a custom port, could not stop?

Screenshot 2024-07-22 at 09 54 25

**Review by louis-jan — APPROVED** [review]
LGTM

**Pull request merged**